refinery/core/models.py

```diff
@@ -2120,9 +2118,9 @@ def get_csv_row(self, aggregates=False):
         def get_aggregate_sum(field_name):
             if not aggregates:
                 return getattr(self, field_name)
-            return SiteStatistics.objects.filter(
+            return list(SiteStatistics.objects.filter(
```

Comment:
Probably unnecessary

Reply:
this is necessary - you cannot index a `dict_value` object

Reply:
and if you need a list just remove `[0]`:
```python
>>> SiteStatistics.objects.all().aggregate(Sum('datasets_shared')).values()
[230]
```

Reply:
That returns a dictionary - I think we need the value of the dictionary, specifically the first one to return the proper element.